packages/ag-charts-community/src/chart/dom/domManager.ts

```diff
@@ -47,8 +54,10 @@ const STYLES = `
 
```

Comment:
Is `ag-charts-canvas-overlay` still used? What does `display: contents` do here?

Reply:
Yes it's still in use - it just skips rendering/calculation/contribution of of margin/padding/border from that element, and just displays contents.

Reply:
Probably need to see it in the browser to follow what's happening. Don't hold off merging, I'll look after it's merged